### PR TITLE
Add check for PowerShell < 4.0

### PR DESCRIPTION
**Pull request by AHOHNMYC**
I had too old PowerShell, and waste almost hour to investigate the root of troubles.
Maybe this PR helps other.

### CODE REVIEW

File: buildenv/1.3.x/win32-static/setup/earlyfetch.ps1 [hunk 1]
```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/env mumble-build
-# Copyright 2013-2017 The 'mumble-releng' Authors. All rights reserved.
+# Copyright 2013-2019 The 'mumble-releng' Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that
 # can be found in the LICENSE file in the source tree or at
 # <http://mumble.info/mumble-releng/LICENSE>.
 
+if ($PSVersionTable.PSVersion.Major -lt 4) {
+	Write-Warning 'This script uses cmdlet Get-FileHash.
+Please, upgrage your Windows Management Framework: https://microsoft.com/download/40855'
```
Comment (davidebeatrici) [inline buildenv/1.3.x/win32-static/setup/earlyfetch.ps1]: Typo: `upgrage` -> `upgrade`.

File: buildenv/1.4.x/win32-static/setup/earlyfetch.ps1 [hunk 2]
```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/env mumble-build
-# Copyright 2013-2017 The 'mumble-releng' Authors. All rights reserved.
+# Copyright 2013-2019 The 'mumble-releng' Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that
 # can be found in the LICENSE file in the source tree or at
 # <http://mumble.info/mumble-releng/LICENSE>.
 
+if ($PSVersionTable.PSVersion.Major -lt 4) {
+	Write-Warning 'This script uses cmdlet Get-FileHash.
+Please, upgrage your Windows Management Framework: https://microsoft.com/download/40855'
```
Comment (davidebeatrici) [inline buildenv/1.4.x/win32-static/setup/earlyfetch.ps1]: Typo: `upgrage` -> `upgrade`.

File: buildenv/1.3.x/win32-static/setup/earlyfetch.ps1 (hunk 1 above)
Comment (AHOHNMYC) [reply buildenv/1.3.x/win32-static/setup/earlyfetch.ps1]: Fixed.

**Comment by davidebeatrici** [comment]
Could you squash the commits, please?

**Comment by AHOHNMYC** [comment]
Uhm, what I supposed to do? Doesn't GitHub has feature `Squash and merge` like GitLab?

**Comment by davidebeatrici** [comment]
Yes, but we usually create a merge commit.

Unfortunately the `Squash and merge` feature puts the commits directly in the `master` branch without it.

You can squash the two commits with `git rebase -i HEAD~2`:
1. Change `pick` with `squash` or `s` at the beginning of the second commit's line, save and exit the editor.
2. Delete the second commit's message from the squashed commit's message, save and exit the editor.
3. Force-push.

**Comment by AHOHNMYC** [comment]
Done! :smiley_cat:

**Comment by davidebeatrici** [comment]
Thank you very much for your contribution!

**Pull request merged**